content/operate/oss_and_stack/stack-with-enterprise/modules-lifecycle.md

```diff
@@ -11,7 +11,7 @@ weight: 7
 ---
 Redis Enterprise Software follows the [Redis Enterprise lifecycle]({{< relref "/operate/rs/installing-upgrading/product-lifecycle" >}}).  (For complete details, see the Redis Enterprise Software [subscription agreement](https://redis.com/software-subscription-agreement).)
 
-The modules included in Redis Stack also follow a release lifecycle and schedule. Here, you'll find the "end-of-life" dates for each module and release.
+The modules included in Redis Community Edition (CE), formerly called Redis Stack, also follow a release lifecycle and schedule. Here, you'll find the "end-of-life" dates for each module and release.
```

Comment:
Instead of "Redis Community Edition formerly called Redis Stack", we should say that we consolidated Redis, modules, and Redis Stack into Redis Community Edition.

Suggesting to replace "(CE)" to "(Redis CE)".